index.js

```diff
@@ -563,7 +563,8 @@ function buildObject (context, location) {
   functionCode += `
     // ${schemaRef}
     function ${functionName} (input) {
-      const obj = ${toJSON('input')}
+      const obj = ${toJSON('input')} || {}
```

Comment:
I would use direct check for null. This will resolve it for any nullish value: 0, "", false, undefined ...

Reply:
Example: https://github.com/fastify/fast-json-stringify/blob/6de8c6cca0695073d29697d5042e2910f8630273/lib/serializer.js#L69